pkg/utils/utils.go

```diff
@@ -188,6 +188,18 @@ func GetSubscription(connection *sdk.Connection, key string) (subscription *amv1
 	return
 }
 
+func GetOrganization(connection *sdk.Connection, key string) (*amv1.Organization, error) {
```

Comment:
I think that it would be nice to add something like
```go
// GetOrganization returns an *amv1.Organization given an OCM cluster name, external id, or internal id as key
```

At least, it isn't clear to me, when reading this code, what I can safely supply to `key string`